ddpui/api/charts_api.py

```
@@ -0,0 +1,599 @@
+"""Chart API endpoints"""
+from typing import Optional, List
+from datetime import datetime, timedelta
```

Ruff (0.12.2)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)